iidm/iidm-xml-converter/src/main/resources/xsd/iidm.xsd

```diff
@@ -20,6 +20,7 @@
     <xs:element name="network">
         <xs:complexType>
              <xs:sequence>
+                 <xs:element name="property" type="iidm:Property" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
Like in the Java model, we also could have network inherit from identifiable?

Reply:
It's not possible to set the network name in the Java model (through factory), so if I make an inheritance in the XSD, there is something to fix in the Java API too. Do you want I change this too?

Reply:
The network factory would not be the best place for setting an optional attribute like name but it is surprising that there is no setName method in `Identiable` interface and that nobody has ever ask for it. 
We can keep it as it is and see later, it is really a minor design issue.